compiler/passes/buildDefaultFunctions.cpp

```diff
@@ -147,23 +151,23 @@ static void buildFieldAccessorFunctions(AggregateType* at) {
   for_fields(field, at) {
     if (!field->hasFlag(FLAG_IMPLICIT_ALIAS_FIELD)) {
       if (isVarSymbol(field)) {
-        build_accessors(at, field);
+        buildAccessors(at, field);
       } else if (isEnumType(field->type)) {
-        build_accessors(at, field);
+        buildAccessors(at, field);
       }
     }
   }
 }
 
 
-// function_exists returns true iff
+// functionExists returns true iff
```

Comment:
Kinda looks like this comment got separated from the function itself?  Definitely not something you've caused, but you are already doing some code clean up in the area